src/test/fuzz/util.h

```diff
@@ -269,8 +269,15 @@ inline CNetAddr ConsumeNetAddr(FuzzedDataProvider& fuzzed_data_provider) noexcep
     const Network network = fuzzed_data_provider.PickValueInArray({Network::NET_IPV4, Network::NET_IPV6, Network::NET_INTERNAL, Network::NET_ONION});
     CNetAddr net_addr;
     if (network == Network::NET_IPV4) {
+        // mingw does not allow designated initializers.
+        #ifdef WIN32
+        in_addr v4_addr_tmp;
+        v4_addr_tmp.S_un.S_addr = fuzzed_data_provider.ConsumeIntegral<uint32_t>();
+        const in_addr v4_addr(v4_addr_tmp);
+        #else
         const in_addr v4_addr = {
             .s_addr = fuzzed_data_provider.ConsumeIntegral<uint32_t>()};
+        #endif
```

Comment:
I don't really like changing code in a build change. Maybe exclude the failing ci config via `--disable-fuzz` for now?

Reply:
So this should probably be `--disable-fuzz-binary` in the windows ci (instead of my suggestion `--disable-fuzz`, which might not go well with how configure is supposed to work)